### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Bupilipili**
### In the branch

-Deleted all the boilerplate from Create React App (text, images, styles).
-Created a new directory called components
-Inside components, created a new Calculator.js file.
-In Calculator.js, created a React component that matches this design: Calculator
-Implemented the design only (JSX and CSS)
-Developed just the UI of the calculator; the math logic to make it work will be added in the next project.
-Used functional components
-Created at least 1 child component

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <button className="button orange" type="button">-</button>
+    <button className="button" type="button">1</button>
+    <button className="button" type="button">2</button>
+    <button className="button" type="button">3</button>
+    <button className="button orange" type="button">+</button>
+    <button className="button zero" type="button">0</button>
+    <button className="button" type="button">.</button>
+    <button className="button orange" type="button">=</button>
+  </div>
+);
```
Comment (AbelG101) [inline src/components/Calculator.js]: - [x] Ouch! You're almost there! However, according to the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md), it's necessary to create at least one child component and incorporate it within your Calculator component. To meet this requirement, kindly consider extracting a portion of the code from this component and placing it in a separate component. Then, utilize that newly created component as the child component here. This adjustment will ensure compliance with the project guidelines. Keep up the good work!

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Bupilipili 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Proper component breakdown ✔
- UI is following proper design guidelines ✔
- Professionally documented readme ✔
- Correct use of Git Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Bupilipili, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Bupilipili) [reply src/components/Calculator.js]: Thank you for the good sighting 🥂 .

**Comment by Bupilipili** [comment]
Ok lets GO 💯

**Pull request merged**